Item.java

```diff
@@ -0,0 +1,20 @@
+package Week_7;
+
+public enum Item {

```

Comment:
In a real world situation, you probably couldn't have this as an enum as you would have to update the code in order to sell a new item.  Imagine if you as the developer would have to send out code updates any time a vending machine operator wanted to sell a new type of candy bar!